src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs

```diff
@@ -0,0 +1,111 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Abstractions.Data;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.SqlServer;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.ChangeFeed
+{
+    /// <summary>
+    /// Data store for resource changes.
```

Comment:
>Data store for resource changes. [](http://example.com/codeflow?start=8&length=32)

It's better to add comment `why` this class needed than summary which just rephrase name of the class.

Reply:
Added a summary.